db/system_keyspace.cc

```diff
@@ -100,6 +100,32 @@ namespace {
             props.enable_schema_commitlog();
         }
     });
+
+    const auto set_group0_table_options =
+        schema_builder::register_static_configurator([](const sstring& ks_name, const sstring& cf_name, schema_static_props& props) {
+            static const std::unordered_set<sstring> tables = {
+                // scylla_local may store a replicated tombstone related to schema
+                // (see `make_group0_schema_version_mutation`), so we use the "repair" GC mode
```

Comment:
"repair" spotted! Hide it from @avikivity, quickly

Reply:
nit: the comments related to the tombstone GC mechanism probably belong to a later commit (the notion of "group 0 table" itself could be considered more generic, in this commit you just introduce that notion)

also this comment assumes that the reader knows the context, i.e., about the tombstone GC mechanism. It might be hard to understand without the context. Perhaps that context should be briefly introduced in the comment itself -- the fact that we use this list of group 0 tables in this mechanism

Reply:
reworded a bit and removed the "repair" keyword